pkg/controller/operators/catalog/operator.go

```diff
@@ -1172,11 +1176,13 @@ func (o *Operator) syncInstallPlans(obj interface{}) (syncError error) {
 		}
 	}
 
+	logger.Infof("Phase coming in: %s", plan.Status.Phase)
 	outInstallPlan, syncError := transitionInstallPlanState(logger.Logger, o, *plan, o.now())
 
 	if syncError != nil {
 		logger = logger.WithField("syncError", syncError)
 	}
+	logger.Infof("Phase coming out: %s", outInstallPlan.Status.Phase)
```

Comment:
ditto - debug logs shouldn't be info

also for this type of thing it's a little nicer to use our structured logs

```go
logger.WithField("phase", outInstallPlan.Status.Phase).Debug("install plan updated")
```